Prace_domowe/Praca_domowa4/Grupa1/Adam_Frej/PD4.ipynb

```diff
@@ -0,0 +1,1479 @@
+{
```

Comment:
<p>Dobrze, że zauważono, że mimo iż metoda SVM nie szacuje prawdopodobieństw przynależności do klas, to jednak można w "pseudosposób" taką informację wyłuskać. Fajnie, że tak wybrano drugi zbiór danych, aby przeanalizować wpływ skalowania zarówno na regresję jak i klasyfikację.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>